Things To Do/Scenes/Home/HomeViewModel.swift

```diff
@@ -19,20 +22,28 @@ final class HomeViewModel: ObservableObject {
     
     init(userDefaultsHelper: UserDefaultsHelper<ToDoItem> = UserDefaultsHelper<ToDoItem>(userDefaultsKey: Constants.toDoItemsKey)) {
         self.userDefaultsHelper = userDefaultsHelper
-        self.toDoItems = loadFromUserDefaults()
-    }
-    
-    private func loadFromUserDefaults() -> [ToDoItem] {
-        userDefaultsHelper.loadData()
+        self.loadData()
```

Comment:
pls call these func in view

Reply:
I'm confused about working with two viewModel in one viewController. I need to read about how can i manage two different screens for same logic.